interface-elements-for-web/articles/report-designer/provide-interactivity/create-drill-down-reports.md

```diff
@@ -19,10 +19,16 @@ Do the following to create a drill-down report:
 	![](../../../images/eurd-web-drilldown-set-drilldown-control.png)
 	
```

Comment:
Please make the following changes above:

Line 15:
`3. Select the [detail report band](..\introduction-to-banded-reports.md), open the **Behavior** category and expand drop-down menu for the band's **Drill-Down Control** property in the [Properties](..\report-designer-tools\ui-panels\properties-panel.md) panel.`

Change "expand drop-down menu" to "expand the drop-down menu".

Line 17:
`This menu displays all report controls available on the report band that is one level above the current band in the report bands' hierarchy.`

Change "that is one level above" to "that are one level above".

Reply:
fixed